commit 6ccce9535e0e6b0749fb8dfa497d07c18c5db3d5
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2019-07-10T18:03:34-07:00 (3 months ago)
    
    TRITON-580 urclient could include some more details about who's running the command
    TOOLS-1308 node-urclient should support automatic reconnection
    TOOLS-1332 ur client does not close HTTP server on close()
    TOOLS-1357 urclient should terminate inflight requests on AMQP abend
    TOOLS-1009 update libuuid to 0.1.4 (urclient)

diff --git a/package.json b/package.json
index 71252cb..13a1a8f 100644
--- a/package.json
+++ b/package.json
@@ -28,7 +28,7 @@
         "json": "9.0.3",
         "jsprim": "0.5.1",
         "tabula": "1.4.1",
-        "urclient": "git+https://github.com/joyent/node-urclient.git#2b9f739",
+        "urclient": "1.2.1",
         "vasync": "1.5.0"
     },
     "devDependencies": {
